addOns/exim/src/main/java/org/zaproxy/addon/exim/log/LogsImporter.java

```diff
@@ -92,16 +91,14 @@ public LogsImporter(File file, LogType type) {
     public LogsImporter(File file, LogType type, ProgressPaneListener listener) {
         this.progressListener = listener;
         success = processInput(file, type);
+        completed();
     }
 
-    private void readModSecLogsFromFile(File file) {
+    private void readModSecLogsFromFile(File file) throws Exception {
         try {
             processModSecLogs(new ModSecurity2AuditReader(file));
         } catch (Exception e) {
-            LOG.warn(
-                    Constant.messages.getString(
-                            ExtensionExim.EXIM_OUTPUT_ERROR, file.getAbsoluteFile()));
-            ExtensionExim.updateOutput(ExtensionExim.EXIM_OUTPUT_ERROR, file.getAbsolutePath());
+            throw (e);
```

Comment:
Was this change on purpose? If so, can just remove the try/catch (same in other places).

Reply:
Yes, and will do.